tests/e2e/utils/config.go

```diff
@@ -62,7 +62,7 @@ func loadConfigJsonFromPath() *Config {
 	return config
 }
 
-//Get config path from Env or hard code the file path
+//getConfigPath to Get config path from Env or hard code the file path
```

Comment:
we can improve the comment like this : 

// getConfigPath returns the configuration path provided in the env var name. In case the env var is not
// set, the default configuration path is returned.

Reply:
@rohitsardesai83 , done the changes of function comments as suggested.